tools/serverpod_cli/test/migrations/migration_generator_exceptions_test.dart

```diff
@@ -39,7 +39,8 @@ void main() {
       test('then null is returned.', () async {
         expect(
           generator.repairMigration(
-            runMode: CreateMigrationCommand.runModes.first /* development */,
+            runMode:
+                CreateRepairMigrationCommand.runModes.first /* development */,
```

Comment:
I really wanted to do that but was discouraged because I wasn't sure how we should handle the serialization if it fails. But I'll give it a stab in a separate commit and then you can take a look to see if it makes sense.

Reply:
Gave it two tries but ended up being scared about changing our public API.

We have "run_modes.dart" exposed from the server also.

Will check later if it is possible to make that API change so that we operator of an enum.